cluster-up/cluster/kind-sriov/sriov-components/manifests/sriovdp-daemonset.yaml

```diff
@@ -26,12 +26,6 @@ spec:
         app: sriovdp
     spec:
       hostNetwork: true
-      nodeSelector:
-        beta.kubernetes.io/arch: amd64
-      tolerations:
-      - key: node-role.kubernetes.io/control-plane
-        operator: Exists
-        effect: NoSchedule
```

Comment:
Is there a scenario on which we need this toleration?
Does it make sense to have different tolerations for the device plugin and CNI?

Reply:
No, I dont think so.
SR-IOV provider support a setup with 1 CP and 2 workers.